docs/user_guide/assessment.rst

```diff
@@ -407,12 +407,55 @@ to take advantage of the inherent plotting capabilities of
 
 .. literalinclude:: ../auto_examples/plot_quickstart.py
     :language: python
-    :start-after: # Analyze metrics using MetricFrame
+    :lines: 32-48
 
 .. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_001.png
     :target: auto_examples/plot_quickstart.html
     :align: center 
 
+It is possible to customize the plots. Here are some common examples:
+
+Customize Plots: `ylim`
+'''''''''''''''''''''''
+The y-axis range is automatically set, which can be misleading, therefore it is
+sometimes useful to set the `ylim` argument to define the yaxis range.
+
+.. literalinclude:: ../auto_examples/plot_quickstart.py
+    :language: python
+    :lines: 51-59
+
+.. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_002.png
+    :align: center
+
+
+Customize Plots: `colormap`
+'''''''''''''''''''''''''''
+To change the color scheme, we can use the `colormap` argument. A list of colorschemes
+can be found `here <https://matplotlib.org/stable/tutorials/colors/colormaps.html>`_.
+
+.. literalinclude:: ../auto_examples/plot_quickstart.py
+    :language: python
+    :lines: 62-70
+
+.. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_003.png
+    :align: center
+
+Customize Plots: `kind`
+'''''''''''''''''''''''
+There are different types of charts (e.g. pie, bar, line) which can be defined by the `kind`
+argument. Here is an example of a bar chart.
+
+.. literalinclude:: ../auto_examples/plot_quickstart.py
+    :language: python
+    :lines: 73-80
+
+.. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_004.png
+    :align: center
+
+There are many other customizations that can be done. More information can be found
+`here <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html>`_ in this
```

Comment:
I think you can link to this directly with 
```
:meth:`pandas.DataFrame.plot`
```
but this is really nitpicky and not a huge improvement. If you want to ignore it just resolve 😄

Reply:
oh that's cool! learning sth new everyday, changed!